core/src/main/java/io/hyperfoil/core/impl/LocalSimulationRunner.java

```diff
@@ -45,7 +46,15 @@ public void run() {
 
       CountDownLatch latch = new CountDownLatch(1);
       init();
-      openConnections(result -> latch.countDown());
+      openConnections(callable -> {
+               try {
+                  callable.call();
```

Comment:
I _think_ this okay as there is no event loop around with a local simulation run, but @franz1981 can you confirm?

Reply:
Will tell you tomorrow, but ATM I would say no

Reply:
Let me check how I could configure block hound for our tests